---
pdg:
    arObjectReader : true
    gazeboObjectReader: false
    OM2MObjectReader: true
    mocapObjectReader: false
    mocapObjectsTopics : '/optitrack/bodies/Rigid_Body_2:/optitrack/bodies/Rigid_Body_1'
    mocapObjectsIds : 'hand:head'
